Shared/Samples/Apply RGB renderer/ApplyRGBRendererView.SettingsView.swift

```diff
@@ -0,0 +1,70 @@
+// Copyright 2025 Esri
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import ArcGIS
+import SwiftUI
+
+extension ApplyRGBRendererView {
+    struct SettingsView: View {
+        /// The action to dismiss the view.
+        @Environment(\.dismiss) private var dismiss
+        /// A binding to the renderer settings.
+        @Binding var settings: RendererSettings
+        
+        var body: some View {
+            Form {
+                Section {
+                    Picker("Type", selection: $settings.stretchType) {
+                        ForEach(StretchType.allCases, id: \.self) { stretchType in
+                            Text(stretchType.label)
+                        }
+                    }
+                }
+                
+                Section {
+                    switch settings.stretchType {
+                    case .histogramEqualization:
+                        EmptyView()
+                    case .minMax:
+                        ColorPicker("Min Color", selection: $settings.minColor, supportsOpacity: false)
+                        ColorPicker("Max Color", selection: $settings.maxColor, supportsOpacity: false)
+                    case .percentClip:
+                        HStack {
+                            Text("\(Int(settings.minValue))")
+                            Spacer()
+                            Text("\(Int(settings.maxValue))")
+                        }
+                        RangeSlider(lowerValue: $settings.minValue, upperValue: $settings.maxValue, range: 0...100, step: 1)
+                    case .standardDeviation:
+                        LabeledContent("Factor", value: settings.standardDeviation, format: .number.precision(.fractionLength(2)))
+                        Slider(value: $settings.standardDeviation, in: 0.0...16.0, step: 0.01) {
```

Comment:
This value doesn't seem to make any noticeable difference when it's > ~6.0? Just wondering why allow all the way up to 16?

Reply:
I had a bit struggle when picked the upper bound.
For a 0 - 255 pixel band value, the biggest possible standard deviation is sqrt(255) = 16. But for most images, since the values will be closer to the center of the histogram, the std will be much smaller (for example, std = 10 can cover a span of 32 - 128). Still, for the most extreme case (all bands are concentrated at either end of 0 or 255), an upper bound 16 will be sufficient.